packages/sdk-codegen/src/sdkModels.spec.ts

```diff
@@ -170,7 +171,179 @@ describe('sdkModels', () => {
+      } as unknown) as OAS.OperationObject
+      const no = ({
+        'x-looker-rate-limited': false,
+      } as unknown) as OAS.OperationObject
+      const nada = {} as OAS.OperationObject
+
+      expect(Method.isRateLimited(yes)).toBe(true)
+      expect(Method.isRateLimited(no)).toBe(false)
+      expect(Method.isRateLimited(nada)).toBe(false)
+    })
+
+    it('X-RateLimit-Limit', () => {
+      const yes = ({
+        'X-RateLimit-Limit': 0,
+      } as unknown) as OAS.OperationObject
+      const no = ({
+        'x-ratelimit-limit': 0,
+      } as unknown) as OAS.OperationObject
+      const nada = {} as OAS.OperationObject
+
+      expect(Method.isRateLimited(yes)).toBe(true)
+      expect(Method.isRateLimited(no)).toBe(false)
+      expect(Method.isRateLimited(nada)).toBe(false)
+    })
+
+    it('X-RateLimit-Remaining', () => {
+      const yes = ({
+        'X-RateLimit-Remaining': 0,
+      } as unknown) as OAS.OperationObject
+      const no = ({
+        'x-ratelimit-remaining': 0,
+      } as unknown) as OAS.OperationObject
+      const nada = {} as OAS.OperationObject
+
+      expect(Method.isRateLimited(yes)).toBe(true)
+      expect(Method.isRateLimited(no)).toBe(false)
+      expect(Method.isRateLimited(nada)).toBe(false)
+    })
+
+    it('X-RateLimit-Reset', () => {
+      const yes = ({
+        'X-RateLimit-Reset': 0,
+      } as unknown) as OAS.OperationObject
+      const no = ({
+        'x-ratelimit-reset': 0,
+      } as unknown) as OAS.OperationObject
+      const nada = {} as OAS.OperationObject
+
+      expect(Method.isRateLimited(yes)).toBe(true)
+      expect(Method.isRateLimited(no)).toBe(false)
+      expect(Method.isRateLimited(nada)).toBe(false)
+    })
+
+    it('X-Rate-Limit-Limit', () => {
+      const yes = ({
+        'X-Rate-Limit-Limit': 0,
+      } as unknown) as OAS.OperationObject
+      const no = ({
+        'x-rate-limit-limit': 0,
+      } as unknown) as OAS.OperationObject
+      const nada = {} as OAS.OperationObject
+
+      expect(Method.isRateLimited(yes)).toBe(true)
+      expect(Method.isRateLimited(no)).toBe(false)
+      expect(Method.isRateLimited(nada)).toBe(false)
+    })
+
+    it('X-Rate-Limit-Remaining', () => {
+      const yes = ({
+        'X-Rate-Limit-Remaining': 0,
+      } as unknown) as OAS.OperationObject
+      const no = ({
+        'x-rate-limit-remaining': 0,
+      } as unknown) as OAS.OperationObject
+      const nada = {} as OAS.OperationObject
+
+      expect(Method.isRateLimited(yes)).toBe(true)
+      expect(Method.isRateLimited(no)).toBe(false)
+      expect(Method.isRateLimited(nada)).toBe(false)
+    })
+
+    it('X-Rate-Limit-Reset', () => {
+      const yes = ({
+        'X-Rate-Limit-Reset': 0,
+      } as unknown) as OAS.OperationObject
+      const no = ({
+        'x-rate-limit-reset': 0,
+      } as unknown) as OAS.OperationObject
+      const nada = {} as OAS.OperationObject
+
+      expect(Method.isRateLimited(yes)).toBe(true)
```

Comment:
What will applications use this `Method.isRateLimited()` for?

Reply:
API Explorer will use it to indicate an endpoint is rate limited.